package.json

```diff
@@ -70,10 +69,8 @@
   },
   "dependencies": {
     "@apollo/client": "^3.5.8",
+    "@bsv/sdk": "^1.0.17",
```

Comment:
I think that there is already newer version :see_no_evil:

Reply:
Yes, you are right -> moved to: @bsv/sdk@1.0.19